packages/legend-art/src/components/Input.tsx

```diff
@@ -19,12 +19,14 @@ import { TimesCircleIcon } from './Icon';
 
 const InputGroup = styled.div`
   position: relative;
+  width: 100%;
```

Comment:
have you tested how this affect query builder projection panel?

Reply:
yes. It was looking the same